File: mau/34277.pas
var a : array[1..1000] of integer;
    n, i, j, dem, k : integer;
BEGIN
    readln(n);
    for i:= 1 to n do read(a[i]);
    for i:= 1 to n do begin
        if a[i] = 0 then continue;
        k:= a[i];
        dem:=0;
        for j:= 1 to n do
            if k=a[j] then begin
                inc(dem);
                a[j]:=0;
            end;
        writeln(k,' ',dem);
    end;
END.

File: mau/34277.cpp
#include <iostream>
using namespace std;
int n, a[1000];
int main()
{
    int i, j, dem, k;
    cin>>n;
    for (i=1; i<= n; i++) cin>>a[i];
    for (i=1; i<=n; i++) {
        if (a[i]==0) continue;
        k = a[i];
        dem = 0;
        for (j=1; j<=n; j++)
            if (k==a[j]) {
                dem++;
                a[j] = 0;
            }
        cout<<k<<" "<<dem<<endl;
    }
    return 0;
}
